sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -4589,6 +4589,20 @@ object SQLConf {
       .stringConf
       .createWithDefault("versionAsOf")
 
+  val RELEASE_ANTLR_CACHE_AFTER_PARSE =
+    buildConf("spark.sql.releaseAntlrCacheAfterParse")
```

Comment:
Please use the existing config namespace, `spark.sql.parser`.
```
- spark.sql.releaseAntlrCacheAfterParse
+ spark.sql.parser.releaseAntlrCacheAfterParse
```

Reply:
Done